File: app/src/main/java/com/neome/java/api/meta/base/dto/StudioEntAutomationStepGenerateImage.java
// neome.ai API - do not change
//

package com.neome.java.api.meta.base.dto;

import com.neome.java.api.meta.base.Types.MetaIdField;
import com.neome.java.api.meta.base.Types.MetaIdForm;
import com.neome.java.api.meta.base.Types.MetaIdVar;

import org.jetbrains.annotations.Nullable;

@SuppressWarnings("unused")
public class StudioEntAutomationStepGenerateImage extends StudioEntAutomationStepGenerateDocument
{
  @Nullable
  public MetaIdField imageFieldId;

  @Nullable
  public MetaIdForm imageFormId;

  @Nullable
  public MetaIdVar imageFormMappingVarId;
}

File: app/src/main/java/com/neome/api/meta/base/dto/StudioEntAutomationStepGenerateImage.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.meta.base.dto

import com.neome.api.meta.base.Types.MetaIdField
import com.neome.api.meta.base.Types.MetaIdForm
import com.neome.api.meta.base.Types.MetaIdVar
import com.neome.api.meta.base.dto.StudioEntAutomationStepGenerateDocument

interface StudioEntAutomationStepGenerateImage : StudioEntAutomationStepGenerateDocument
{
  val imageFieldId: MetaIdField?
  val imageFormId: MetaIdForm?
  val imageFormMappingVarId: MetaIdVar?
}